modules/videopress/class.videopress-gutenberg.php

```diff
@@ -164,6 +165,32 @@ public function render_video_block_with_videopress( $attributes, $content ) {
 			1
 		);
 	}
+
+	/**
+	 * Replaces the video uploaded in the block editor.
+	 *
+	 * Enqueues a script that registers an API fetch middleware replacing the video uploads in Gutenberg so they are
+	 * uploaded against the WP.com API media endpoint and thus transcoded by VideoPress.
+	 */
+	public function override_video_upload() {
+		// Bail if WP.com site.
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			return;
+		}
+
+		// Bail if Jetpack or VideoPress is not active.
+		if ( ! Jetpack::is_active() || ! Jetpack::is_module_active( 'videopress' ) ) {
+			return;
+		}
+
+		wp_enqueue_script(
+			'jetpack-videopress-gutenberg-override-video-upload',
+			plugin_dir_url( __FILE__ ) . 'js/gutenberg-video-upload.js',
```

Comment:
I think you could enqueue the minified version in prod environments, like so:

```diff
diff --git a/modules/videopress/class.videopress-gutenberg.php b/modules/videopress/class.videopress-gutenberg.php
index 156945c69..0b325b35e 100644
--- a/modules/videopress/class.videopress-gutenberg.php
+++ b/modules/videopress/class.videopress-gutenberg.php
@@ -5,6 +5,8 @@
  * @package Jetpack
  */
 
+use Automattic\Jetpack\Assets;
+
 /**
  * Register a VideoPress extension to replace the default Core Video block.
  */
@@ -185,7 +187,10 @@ class VideoPress_Gutenberg {
 
 		wp_enqueue_script(
 			'jetpack-videopress-gutenberg-override-video-upload',
-			plugin_dir_url( __FILE__ ) . 'js/gutenberg-video-upload.js',
+			Assets::get_file_url_for_environment(
+				'_inc/build/videopress/js/gutenberg-video-upload.min.js',
+				plugin_dir_url( __FILE__ ) . 'js/gutenberg-video-upload.js'
+			),
 			array( 'wp-api-fetch', 'wp-polyfill', 'lodash' ),
 			JETPACK__VERSION,
 			false

```

Reply:
Changed in fbbfb53